### PR TITLE
Documentation: add initial benchmarks

### PR DESCRIPTION
**Pull request by kelseyhightower**

### CODE REVIEW

File: Documentation/benchmarks.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+## Physical machines
+
+GCE n1-highcpu-2 machine type
+
+- 1x dedicated local SSD mounted under /var/lib/etcd
+- 1x dedicated slow disk for the OS
+- 1.8 GB memory
+- 2x CPUs
+
```
Comment (xiang90) [inline Documentation/benchmarks.md]: version of etcd?

File: Documentation/benchmarks.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+## etcd Cluster
+
+3 etcd members, each runs on a single machine
+
+## Testing
+
+Bootstrap another machine and use benchmark tool to send requests to etcd cluster.
```
Comment (xiang90) [inline Documentation/benchmarks.md]: we do not have a useable benchmark to share yet.

**Comment by xiang90** [comment]
@kelseyhightower How are we going to keep history of the benchmark result? i suggest to put it under /doc/bench/2.1.0

**Comment by kelseyhightower** [comment]
@xiang90 Yeah. Let me move this.

**Comment by xiang90** [comment]
LGTM

**Pull request merged**